SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -108,7 +108,30 @@ void ComponentReader::ApplySchemaObject(Schema_Object* ComponentObject, UObject*
 
 	TArray<UProperty*> RepNotifies;
 
-	for (uint32 FieldId : UpdatedIds)
+	TArray<Schema_FieldId> InitialIds;
+	if (UNLIKELY(bIsInitialData))
```

Comment:
Poll: react with 👍  or 👎 if you like the unlikely macro.

Reply:
I feel we shouldn't do things like this until we've at least tried to measure something or refactor code to not have the problem in the first place.